### PR TITLE
Tweaks to Topology Manager docs PR

### PR DESCRIPTION
**Pull request by sftim**
@lmdaly here are some suggestions for https://github.com/kubernetes/website/pull/15716

Feel free to cherry-pick or revise

### CODE REVIEW

File: content/en/docs/concepts/architecture/nodes.md [hunk 1]
```diff
@@ -291,6 +291,13 @@ includes all containers started by the kubelet, but not containers started direc
 If you want to explicitly reserve resources for non-Pod processes, follow this tutorial to
 [reserve resources for system daemons](/docs/tasks/administer-cluster/reserve-compute-resources/#system-reserved).
 
+## Node topology
+
+{{< feature-state state="alpha" >}}
+
+If you have enabled the `TopologyManager`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) then
+the kubelet can pay attention to topology hints.
```
Comment (lmdaly) [inline content/en/docs/concepts/architecture/nodes.md]: Maybe something like "the kubelet can use topology hints to when making resource assignment decisions"

File: content/en/docs/concepts/configuration/assign-pod-node.md [hunk 2]
```diff
@@ -397,4 +397,8 @@ The design documents for
 [node affinity](https://git.k8s.io/community/contributors/design-proposals/scheduling/nodeaffinity.md)
 and for [inter-pod affinity/anti-affinity](https://git.k8s.io/community/contributors/design-proposals/scheduling/podaffinity.md) contain extra background information about these features.
 
+Once a Pod is assigned to a Node, the kubelet runs the Pod and allocates node-local resources.
+The [topology manager](/docs/tasks/administer-cluster/topology-manager/) takes part in node-level
+scheduling decisions.
```
Comment (lmdaly) [inline content/en/docs/concepts/configuration/assign-pod-node.md]: Suggestion: takes part in node-level resource allocation decisions?

File: content/en/docs/concepts/scheduling/kube-scheduler.md [hunk 3]
```diff
@@ -107,7 +107,7 @@ kube-scheduler has a default set of scheduling policies.
   volumes should be attached, and whether that's over a configured limit.
 
 - `CheckNodeMemoryPressure`: If a Node is reporting memory pressure, and there's no
-  configured exception, the Pod won't be scheduled there.
+  configured Gxception, the Pod won't be scheduled there.
```
Comment (lmdaly) [inline content/en/docs/concepts/scheduling/kube-scheduler.md]: I guess this change is not meant to be in here :)

**Review by lmdaly — COMMENTED** [review]
Looks overall good, thanks for the changes!
Couple of small suggestions and I can merge PR (might need a rebase I just pushed some changes)

**Comment by sftim** [comment]
Tweaked; rebased; squashed; pushed

**Review by lmdaly — APPROVED** [review]
LGTM!

**Comment by lmdaly** [comment]
> Tweaked; rebased; squashed; pushed

Thanks @sftim. Merged!

**Pull request merged**